src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java

```diff
@@ -151,4 +151,38 @@ private static JWTVerifier getVerifier(Request req, String token) {
     public static boolean authDisabled() {
         return hasConfigProperty("DISABLE_AUTH") && "true".equals(getConfigPropertyAsText("DISABLE_AUTH"));
     }
+
+    /**
+     * Confirm that the user exists
+     */
+    private static Auth0UserProfile isValidUser(Request request) {
+
+        Auth0UserProfile profile = getUserFromRequest(request);
+        if (profile == null || (profile.adminUser == null  && profile.otpUser == null && profile.apiUser == null)) {
+            logMessageAndHalt(request, HttpStatus.NOT_FOUND_404, "Unknown user.");
+        }
+
+        return profile;
+    }
+
+    /**
+     * Confirm that the user's actions are on their items if not admin.
+     */
+    public static void isAuthorized(String userId, Request request) {
```

Comment:
Let's make sure this isn't duplicating what's been added by #14.

Reply:
Pretty sure this is ok. https://github.com/ibi-group/otp-middleware/pull/14 appears to be checking if the user is admin. This method authorizes admins and users if they original created the item they are requesting.